repos/system_upgrade/common/actors/udev/udevadminfo/tests/test_udevadminfo.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from leapp.libraries.actor import udevadminfo
+from leapp.libraries.common import testutils
+from leapp.libraries.stdlib import api, run
+from leapp.models import UdevAdmInfoData
+
+CUR_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+def test_udevadminfo(monkeypatch):
+
+    with open(os.path.join(CUR_DIR, 'files', 'udevadm_database'), 'r') as fp:
+        mocked_data = fp.read()
+    monkeypatch.setattr(api, 'produce', testutils.produce_mocked())
+    monkeypatch.setattr(udevadminfo, 'run', lambda *args: {'stdout': mocked_data})
+    udevadminfo.process()
+
+    assert api.produce.called == 1
+    assert isinstance(api.produce.model_instances[0], UdevAdmInfoData)
+    assert api.produce.model_instances[0].db == UdevAdmInfoData(db=mocked_data).db
```

Comment:
It is a bit weird to create an `UdevAdmInfoData` instance in this case. 
```suggestion
    assert api.produce.model_instances[0].db == mocked_data
```